wordpress-mobile/AztecEditor-iOS
# Fixes some issues with lists.

### Description:

This PR fixes #456.

Lists were losing some of their formatting when combined with multi-line selection and newlines in general.

This PR solves those issues.

### Testing:

**Test 1:**

1. Launch the empty editor.
2. Add an ordered list with 3 lines of text or more.
3. Select part of the list.
4. Convert to unordered list.
5. Switch to HTML and back.  Make sure everything is OK.

**Additional testing:**

Play with lists and try to break them in visual mode.
